modules/acquia_cms_common/tests/src/FunctionalJavascript/Ckeditor5ConfigurationTestBase.php

```diff
@@ -29,6 +29,7 @@ abstract class Ckeditor5ConfigurationTestBase extends WebDriverTestBase {
    */
   protected static $modules = [
     'acquia_cms_common',
+    'acquia_cms_image',
```

Comment:
@deepakmishra2 Why this has been added here ?

Reply:
@vishalkhode1 added by Ankit, he has removed that now along with media_library module.